owner.go

```diff
@@ -23,3 +23,15 @@ func (s *EntityOwnerTeam) AsTeam() TeamId {
 		Id:    s.Id,
 	}
 }
+
+type EntityOwnerService struct {
+	OnService ServiceId `graphql:"... on Service"`
+}
+
+func (s *EntityOwnerService) Aliases() []string {
+	return s.OnService.Aliases
+}
+
+func (s *EntityOwnerService) Id() ID {
+	return s.OnService.Id
+}
```

Comment:
I don't love this - theres probably a better way - but lets ship it for now because its not a one way door to fix.